ord_schema/__init__.py

```diff
@@ -37,7 +37,7 @@
     reaction_pb2.PressureConditions.Atmosphere,
     reaction_pb2.PressureConditions.PressureMeasurement,
     reaction_pb2.PressureConditions.PressureControl,
-    reaction_pb2.ProductCompound.Texture,
+    reaction_pb2.Texture,
```

Comment:
keep this list alphabetical

Reply:
fixed in https://github.com/open-reaction-database/ord-schema/pull/698/commits/c03e02687ae6831084669afc9a01eee6e3984d55